README.md

```diff
@@ -1,14 +1,16 @@
 
 
+
 # GitHub Stats
+A simple project present GitHub projects download static regarding of their desire OS.
 
 Please check deployed version
-[Preview](https://spread-the-code.github.io/github-stats)
+[Preview](https://github-state.netlify.app/)
 
-![screen record of preview](https://spread-the-code.github.io/github-stats/blob/master/preview01.png)
+![screen record of preview](https://github.com/spread-the-code/github-stats/blob/master/preview01.png)
```

Comment:
I think that you need to use the raw version: https://raw.githubusercontent.com/spread-the-code/github-stats/master/preview01.png

<img src="https://raw.githubusercontent.com/spread-the-code/github-stats/master/preview01.png" />